test/service-start.bats

```diff
@@ -71,15 +71,3 @@
   echo "$output"
   [[ $output = *NotFound* ]]
 }
-
-@test "examples: roundtrip Z-TDF" {
-  go run ./examples encrypt "Hello Zero Trust"
-  go run ./examples decrypt sensitive.txt.tdf
-  go run ./examples decrypt sensitive.txt.tdf | grep "Hello Zero Trust"
-}
-
-@test "examples: roundtrip nanoTDF" {
-  go run ./examples encrypt -o sensitive.txt.ntdf --nano "Hello NanoTDF"
-  go run ./examples decrypt sensitive.txt.ntdf
-  go run ./examples decrypt sensitive.txt.ntdf | grep "Hello NanoTDF"
-}
```

Comment:
Some people are using this as a reference.  Can we keep it?  or leave some comment

Reply:
These are moved over to tdf-roundtrips.bats. I've refactored that file to be easier to read (moving helper methods to the end instead of the start) and added a note at the start of this file that it is just for tests that make sure the system if running and exposing the expected featuers.